cogs/surveys.py

```diff
@@ -57,11 +61,6 @@ async def on_message(self, message: discord.Message) -> None:
                             )
                         break
         await self.bot.process_commands(message)
-        with suppress(AttributeError):
-            Log(
-                f" $ sent a survey in {message.channel.name}, bot responded",
-                message.author,
-            )
 
```

Comment:
Do we really want to delete the log altogether? I feel like it would be more appropriate to keep the log, but move it so that it only fires when a survey link is detected.

Reply:
Correct, it still logs when a survey is posted and the cog is used. We just had 2 logging blocks in the survey.py and the one at the end ran everytime.

Reply:
Ah, okay. Didn't see the other logging block, thanks.